pandas/core/generic.py

```diff
@@ -3636,6 +3637,50 @@ def astype(self, dtype, copy=True, errors='raise', **kwargs):
         Returns
         -------
         casted : type of caller
+
+        Examples
+        --------
+        >>> ser = pd.Series([1, 2], dtype='int32')
+        >>> ser
+        0    1
+        1    2
+        dtype: int32
+        >>> ser.astype('int64')
+        0    1
+        1    2
+        dtype: int64
+
+        Convert to pd.Categorial:
```

Comment:
Can you write something like "convert to categorical type" or "categorical Series" (as it does not return a pd.Categorical object, but a Series with categorical dtype)